src/main/scala/cromwell/binding/expression/Evaluator.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding.expression
+
+import cromwell.parser.WdlParser.AstNode
+
+import scala.language.postfixOps
+import scala.util.Try
+
+trait Evaluator {
+  type T
```

Comment:
Oh sure, when info this you tell me to give a full name :)

The scala idiom is to start with A, not T - but for more complicated things to use UpperCamelCase